README.md

```diff
@@ -1 +1,42 @@
-# To-do-list
\ No newline at end of file
+# To-Do List
+
+> I am creating a to-do list for a personal project.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://benjp009.github.io/To-do-list/)
+
+## Getting Started
+
+**Feel free to discover the code and help us improve it**
+
```

Comment:
- [OPTIONAL] Please  it will be nice to add steps to clone and run your project locally.

Reply:
Will do 
thanks